docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -298,6 +301,91 @@ To log out of the aws application and remove credentials:
 $ tsh app logout awsconsole-test
 ```
 
+## Step 9/9. Access applications using AWS SDKs
+
+First, log into the previously configured console app if you haven't already
+done so.
+
+```code
+$ tsh app login --aws-role ExamplePowerUser awsconsole-test
+```
+
+To connect an application that uses AWS SDKs, you can start either a local
+HTTPS proxy or a local AWS Service Endpoint proxy.
+
+<Tabs>
+  <TabItem label="HTTPS proxy">
+  By default, starting the AWS app proxy creates a local HTTPS proxy server
+  that forwards AWS requests to the Teleport Proxy Service, enabling you to
+  access AWS applications.
+
+  Now, use the following command to start the proxy your applications will be
+  connecting to:
+
+  ```code
+  $ tsh proxy aws -p 23456
+  Started AWS proxy on http://127.0.0.1:23456.
+
+  Use the following credentials and HTTPS proxy setting to connect to the proxy:
+    AWS_ACCESS_KEY_ID=<access-key-id>
+    AWS_SECRET_ACCESS_KEY=<secret-access-key>
+    AWS_CA_BUNDLE=<ca-bundle-path>
+    HTTPS_PROXY=http://127.0.0.1:23456
+  ```
+
+  Use the displayed AWS credentials and HTTPS proxy settings when configuring
+  your application.
+
+  It is important to check how AWS credentials and HTTPS proxy setting can be
+  configured for your application. For example, many command line tools like
+  `terraform` or `eksctl` supports setting the above AWS credentials and the
+  HTTPS proxy using environment variables:
+
+  ```code
+  $ export AWS_ACCESS_KEY_ID=<access-key-id>
+  $ export AWS_SECRET_ACCESS_KEY=<secret-access-key>
+  $ export AWS_CA_BUNDLE=<ca-bundle-path>
+  $ export HTTPS_PROXY=http://127.0.0.1:23456
+  $ terraform
+  ```
+
+  If you are developing your own applications using AWS SDKs, some SDKs may
+  require extra environment variables (e.g. `AWS_SDK_LOAD_CONFIG=true` for AWS
+  SDK for Go v2) or require configuring HTTPS proxy through code (e.g. AWS SDK
+  for JavaScript).
+  </TabItem>
+
+  <TabItem label="AWS Service Endpoint proxy">
+  If the application cannot use a HTTPS proxy, start the AWS app proxy with the
+  `--endpoint-url` flag to create a local server that can be used as an AWS
+  Service Endpoint.
+
+  ```code
+  $ tsh proxy aws --endpoint-url -p 23457
+  Started AWS proxy which serves as an AWS endpoint URL at https://localhost:23457
+
+  In addition to the endpoint URL, use the following credentials to connect to the proxy:
+    AWS_ACCESS_KEY_ID=<access-key-id>
+    AWS_SECRET_ACCESS_KEY=<secret-access-key>
+    AWS_CA_BUNDLE=<ca-bundle-path>
+  ```
+
+  Many tools support overwriting the AWS Service Endpoints through command
+  flags or environment variables. For example, to use the AWS Service Endpoint
+  with AWS CLI:
+  ```code
+  $ export AWS_ACCESS_KEY_ID=<access-key-id>
+  $ export AWS_SECRET_ACCESS_KEY=<secret-access-key>
+  $ export AWS_CA_BUNDLE=<ca-bundle-path>
+  $ aws s3 ls --endpoint-url https://localhost:23457
+  ```
```

Comment:
This example looks a bit artificial - not sure what is the use case of `aws s3 ls --endpoint-url` if we are providing `tsh aws` command. Can we provide other use case when `--endpoint-url` is really needed ?

Reply:
Yes. I "shopped" around but couldn't find some a good example (application) for this. I am leaning towards removing this special "Service Endpoint" mode in this guide. Alternatively, i can copy the `dynamodb-admin` from the DynamoDB guide. What do you think?

Reply:
Removed the service endpoint url mode section =D

Reply:
Friendly ping @smallinsky thanks!